app/assets/javascripts/vocab_bar.js

```diff
@@ -5,12 +5,12 @@ $(document).ready(function() {
     var offset = searchResultBanner.offset().top;
     var bottom = offset + searchResultBanner.outerHeight(true);
     var baseMargin = 24; // 1.5rem
+    var barBottom = bar.offset().top + bar.outerHeight(true) + baseMargin;
     var initialOffset = bottom + baseMargin;
     var footerTop = $('.sticky_footer').offset().top;
 
     function positionVocabBar() {
       var windowScroll = $(window).scrollTop();
-      var barBottom = bar.offset().top + bar.outerHeight(true) + baseMargin;
```

Comment:
I've checked this locally and it looks like shifting this variable out of the method has caused it to allow for the sidebar to hide behind the footer now.
![screen shot 2018-09-03 at 10 46 03 am](https://user-images.githubusercontent.com/33815339/44961549-f531b280-af66-11e8-88c0-50d973cb9d88.png)

Reply:
Gaaah, okay.